scipy/stats/_continuous_distns.py

```diff
@@ -8402,14 +8402,12 @@ class truncnorm_gen(rv_continuous):
 
     Notes
     -----
-    The standard form of this distribution is a standard normal truncated to
-    the range ``[a, b]``, where ``a`` and ``b`` are user-provided shape
-    parameters. The parameter ``loc`` shifts the mean of the underlying normal
-    distribution, and ``scale`` controls the standard deviation of the
-    underlying normal, but ``a`` and ``b`` are still defined with respect to
-    the *standard* normal. If ``myclip_a`` and ``myclip_b`` are clip values
-    defined with respect to a shifted and scaled normal, they can be converted
-    the required form according to::
+    This distribution is the normal distribution centred on ``loc`` (default
```

Comment:
```suggestion
    This distribution is the normal distribution centered on ``loc`` (default
```
This is the more common spelling in the codebase:
![image](https://user-images.githubusercontent.com/6570539/187726994-66ead906-09cd-4a08-81bb-6f667bb246c8.png)

vs
![image](https://user-images.githubusercontent.com/6570539/187726924-a2b55c0f-da88-4bd3-9f86-a44cbf6524d9.png)